test/serializers/non_paginated_collection_serializer_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    class NonPaginatedCollectionSerializerTest < ActiveSupport::TestCase
+      setup do
+        posts = [Post.new(id: 1)]
+        @serializer = NonPaginatedCollectionSerializer.new(posts)
+      end
+
+      def test_inherit_from_collection_serializer
+        assert(NonPaginatedCollectionSerializer < CollectionSerializer)
```

Comment:
Not sure if this is the best way to test that this class is a subclass of `CollectionSerializer` (and there behave the same) or if I should write some "shared test" via a module and include it in here.